styleguide/derek/pattern-components/challenges-river/_challenges-river.scss

```diff
@@ -0,0 +1,87 @@
+.challenges-wrapper {
+  margin-top: 50px;
+}
+
+.challenges-header,
+.challenges-subhead {
+  margin: 0;
+  margin-bottom: 25px;
+  padding-left: 25%;
+}
+
+.challenges-itemWrapper {
+  @include make-xs-column(6);
+
+  padding: 0;
+
+  :nth-child(odd) {
+    .challenges-item {
+      float: right;
+    }
+  }
+}
+
+.challenges-item {
+  @include make-xs-column(10);
+  @include make-md-column(8);
+
+  margin-bottom: 50px;
+
+  .rackshape {
```

Comment:
Instead of nesting here can we not just define a top-level class called `.challenges-itemRackShape` or something like that?